src/Input/Input.js

```diff
@@ -120,15 +120,24 @@ class Input extends React.PureComponent {
       type,
     } = this.props
     const { showPassword } = this.state
-    const inputType = (type === 'password' && showPassword)
+    let inputType = (type === 'password' && showPassword)
       || multiline
       ? 'text'
       : type
 
+    inputType = inputType === 'phone'
+      ? 'phone'
```

Comment:
A ideia aqui é converter o nosso `phone` para `tel`, certo? Na story de base `dark` só funcionou corretamente no Android depois que mudei esta linha para:

```suggestion
      ? 'tel'
```